File: 4/Shirt.py
class Shirt:
    def __init__(self) -> None:
        self.Id = 0
        self.Color = ""
        self.Design = ""

    def to_dict(self):
        return {
            'Id': self.Id,
            'Color': self.Color,
            'Design': self.Design
        }

File: 8/ShirtsShop/Shared/Shirt.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ShirtsShop.Shared
{
    public class Shirt
    {

        public int Id { get; set; } = 0;

        public string Color { get; set; }

        public string Text { get; set; }

        public Size Size { get; set; }

        public int SizeId { get; set; }
    }
}